### PR TITLE
use new resource router implementation

### PR DESCRIPTION
**Pull request by xh3b4sd**
Towards https://github.com/giantswarm/operatorkit/pull/147.

### CODE REVIEW

File: service/kvmconfig/framework.go [hunk 1]
```diff
@@ -144,22 +177,43 @@ func NewFramework(config FrameworkConfig) (*framework.Framework, error) {
...
-		return ctx, nil
+	var v3ResourceSet *framework.ResourceSet
+	{
+		c := framework.ResourceSetConfig{}
+
+		c.Handles = v3ResourceSetHandles
+		c.InitCtx = v3InitCtx
+		c.Logger = config.Logger
+		c.Resources = v3Resources
+	}
```
Comment (kopiczko) [inline service/kvmconfig/framework.go]: I thought we can provide ResourceSets inside v3 -like packages. That would make imports easier.

Comment (kopiczko) [reply service/kvmconfig/framework.go]: But it's ok for the time being. I can propose something later.

Comment (xh3b4sd) [reply service/kvmconfig/framework.go]: There are probably further improvements we can make. I agree. Lets iterate from here.

**Comment by xh3b4sd** [comment]
Successfully tested on gorgoth.

**Pull request merged**